packages/web/src/scss/components/Collapse/_Collapse.scss

```diff
@@ -0,0 +1,32 @@
+@use 'sass:map';
+@use '../../tools/breakpoint';
+@use 'theme';
+
+.Collapse {
+    position: relative;
+    overflow: hidden;
+    height: 0;
+    transition: height 0.25s cubic-bezier(0.4, 0, 0.2, 1) 0s;
+
+    // &.is-collapsed{}
+
+    &[data-breakpoint='mobile'] {
```

Comment:
Can you loop over breakpoints so it is more dynamic?

Reply:
Done